plans/rpmlint/README.md

```diff
@@ -0,0 +1,56 @@
+# Rpmlint
+
+<!-- SPHINX-START -->
+
+Run [rpmlint] on the current Copr project or Koji build
```

Comment:
Can we maybe briefly describe what rpmlint does?

Reply:
Hmm, I am not sure myself. Some things I've noticed it does is to check if we are packaging empty files, dangling symlink, etc. I think there is also some overlap between `rpmlint` and `rpminspect`

Reply:
I was thinking about that, but their description is not very useful:

> Tool for checking common errors in rpm packages 
> 
> rpmlint is a tool for checking common errors in RPM packages. rpmlint can be used to test individual packages before uploading or to check an entire distribution.

What do you think?

Reply:
I quite like the next sentence:

> rpmlint can check binary RPMs, source RPMs, and plain specfiles, but all checks do not apply to all argument types. For best check coverage, run rpmlint on source RPMs instead of plain specfiles.

I think we should make clear what we are actually running rpmlint on.

Reply:
That I am having in the section just below. Should we include that section in the middle there, or do you prefer a more verbose one-liner there at the top? I was writing it so that the top part is the brief summary, while the rest of the details I put down in `## Description`

Reply:
OK, let's leave it as is then.